schema/valid.js

```diff
@@ -8,7 +8,5 @@ module.exports = function validate(data) {
 
 	if(!valid) {
 		throw new Error("Your ExtractTextPlugin config is not correct. Please double check.");
```

Comment:
You can get a string message out of ajv. See https://github.com/webpack-contrib/purifycss-webpack for the basic idea. I do also defaults there.